uPortal-webapp/src/main/webapp/media/skins/respondr/common/less/gallery.less

```diff
@@ -30,84 +30,160 @@
  * $dgraydark=@grayscale8 | $dgraydarker=@grayscale9 | $dgraydarkest=@grayscale10
  * Colors
  */
-@galleryBackground              : @grayscale10;
-@galleryBorder                  : 1px solid @grayscale4;
-@galleryHandleBackground        : @black url("./common/images/gallery-bar.png") bottom left repeat-x;
-@galleryHandleBorder            : 1px solid @grayscale7;
-@galleryHandleLinkBackround     : @white scroll url("./common/images/gallery-handle.png") 0 0 repeat-x;
-@galleryHandleLink              : @grayscale9;
-@galleryHandleRound             : 3px;
-@galleryHandleShadow            : 0 1px 2px @grayscale4;
-@galleryHandleIconClosed        : url("./common/images/gallery-handle-arrow-down.png");
-@galleryHandleIconOpen          : url("./common/images/gallery-handle-arrow-up.png") !important;
-@galleryBarBackground           : transparent url("./common/images/gallery-bar.png") top left repeat-x;
-@galleryMenuBackground          : @grayscale9;
-@galleryMenuItemBackground      : @grayscale8;
-@galleryMenuItemLink            : @grayscale3;
-@galleryMenuItemBackgroundHover : @color1-darker;
-@galleryMenuItemLinkHover       : @white;
+@gallerybackground : @grayscale10;
```

Comment:
prettier can be disabled for this section if we want to preserve the original styling